app/views/publishers/vacancies/_publish_buttons.html.slim

```diff
@@ -0,0 +1,10 @@
+- if vacancy.draft? && vacancy.errors.none?
+  h3.govuk-heading-m = t("jobs.preview_listing.heading")
+  p.govuk-body = t("jobs.preview_listing.message")
+  = govuk_button_link_to t("buttons.preview_job_listing"), organisation_job_preview_path(vacancy.id), class: "preview-listing-gtm", id: "vacancy-review-preview"
+
+  h3.govuk-heading-m = t("jobs.submit_listing.heading", count: vacancy.publish_on.today? ? 0 : 1, date: vacancy.publish_on.to_s.strip)
```

Comment:
I can't see how the `vacancy.publish_on` nil bug is accounted for here...

Reply:
it's addressed on first line:

`- if vacancy.draft? && vacancy.errors.none?`